app/packs/src/apps/mydb/elements/list/ElementsTable.js

```diff
@@ -269,6 +295,60 @@ export default class ElementsTable extends React.Component {
     );
   };
 
+  attachmentTokenModal = () => {
+    const { showAttachmentTokenModal, attachmentTokens, showEmpty } = this.state;
+    const { currentElement } = ElementStore.getState();
+    return (
+      <Modal show={showAttachmentTokenModal} onHide={this.toggleAttachmentTokens}>
+        <Modal.Header closeButton />
+        <Modal.Body>
+          <Panel>
+            <Panel.Heading>
+              <Panel.Title>
+                List of Attachment Tokens for element {currentElement?.name}:
+              </Panel.Title>
+            </Panel.Heading>
+            <Panel.Body>
+              {
+                attachmentTokens?.map((item, idx) => {
+                  const key = Object.keys(item);
+                  const value = item[key[0]];
+                  const key_split = key[0].split("/");
+                  return currentElement?.id == key_split[3] && (
```

Comment:
Is there any particular reason for using == instead of === here?

Reply:
Yes, type conversion is required here!